inference-engine/include/ie_transformations.hpp

```diff
@@ -52,5 +52,40 @@ namespace InferenceEngine {
  * @param network A network to apply LowLatency transformation
  * *
  */
+
+INFERENCE_ENGINE_DEPRECATED("Use InferenceEngine::LowLatency2 instead")
```

Comment:
Please extend this message. Do we know the date when we will remove this API?

Reply:
Date of deletion was not discussed

Reply:
@ilyachur  This transformation will be removed in 2023.1.